### ISSUE TITLE
Rule lines-around-comment not triggering at begin of file

### ISSUE THREAD

**Issue by ReiKratos** [eslint/eslint#10831]
* **ESLint Version:** 4.12.1
* **Node Version:** 9.11.1
* **npm Version:** 6.4.1

**Parser:** Babel-eslint

**Configuration:**
```yml
rules:
  lines-around-comment: ["error", {
    afterBlockComment: true,
    beforeBlockComment: true,
    }
  ]
```

With that configuration, the following code is supposed to throw an error:

```js
/*
 * Some comment.
 */
```

However, when block comment starts at the first line of the file, the rule doesn't trigger. Is it on purpose? If it is, why is that?

**Comment by kaicataldo** [comment]
This is [by design](https://github.com/eslint/eslint/blob/master/lib/rules/lines-around-comment.js#L327). The intention of the rule is to create whitespace around comments to improve readability, and the assumption is that a user wouldn't want extra whitespace at the beginning or end of the file because there isn't anything before or after the comment, respectively.

**Comment by kaicataldo** [comment]
Closing this issue as it looks like the question has been answered. Please feel free to visit us in the [ESLint Gitter](https://gitter.im/eslint/eslint) if you have any other issues!
